src/System.Reflection.Metadata/src/System/Reflection/PortableExecutable/PEHeaderBuilder.cs

```diff
@@ -56,6 +65,16 @@ public sealed class PEHeaderBuilder
             ulong sizeOfHeapReserve = 0x00100000,
             ulong sizeOfHeapCommit = 0x1000)
         {
+            if (fileAlignment < 512 || fileAlignment > 64 * 1024 || BitArithmetic.CountBits(fileAlignment) != 1)
```

Comment:
Probably doesn't matter here, but `CountBits(x) == 1` is costlier than necessary for power of 2 check. 

https://graphics.stanford.edu/~seander/bithacks.html#DetermineIfPowerOf2